packages/sampler/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 The following is a curated list of changes in the Enact Sampler, newest changes on the top.
 
+## [unreleased]
+
+### Added
+
+- `ui/ViewManager` sample
```

Comment:
We usually don't leave Changelog when we fix stories.
Let's remove this also

Reply:
remove it